ocaml/xapi/xapi.ml

```diff
@@ -913,6 +913,7 @@ let server_init() =
           (* CA-13878: make sure PBD plugging has happened before attempting to reboot any VMs *)
           "resynchronising VM state", [], (fun () -> Xapi_xenops.on_xapi_restart ~__context);
           "listening to events from xapi", [], (fun () -> if not (!noevents) then ignore (Thread.create Xapi_xenops.events_from_xapi ()));
+          "listening to events from xapi", [], (fun () -> ignore (Thread.create Network_event_loop.watch_networks ()));
```

Comment:
Yep, I'll do that, thanks. I'll still need `Server_helpers.exec_with_new_task`, because this has to have type `unit -> unit`, but I still need a `Context`.